### ISSUE TITLE
Feature: Passwords for Office/PDF Files

### ISSUE THREAD

**Issue by Crashman1983** [cuckoosandbox/cuckoo#227]
I've to analyse some Documents with passwords. It is hard to put in the right password during a hot analysis. 

I couldn't find an option to pass a password from host to guest, except for zip files. 

Maybe this could be implemented.

thx

**Comment by jbremer** [comment]
Anyone that would like to implement this feature?

**Comment by zashraf1337** [comment]
https://github.com/spender-sandbox/cuckoo-modified/issues/414#issuecomment-278958570

will this help?

**Comment by jbremer** [comment]
@zashraf1337 Yes, but it'll have to be integrated into `sflock` (https://github.com/jbremer/sflock) and preferably we find a cross-platform'ish solution.

**Comment by zashraf1337** [comment]
just fya / ref: https://github.com/spender-sandbox/cuckoo-modified/search?utf8=%E2%9C%93&q=msoffice&type=
